Tests/integration/UserLocal.Tests.ps1

```diff
@@ -120,13 +120,201 @@ Describe "Add User Local" {
             $userlocal.'email-to' | Should -Be "powerfgt@power.fgt"
         }
 
+    }
+
+    Context "Local User (RADIUS)" {
+
+        BeforeAll {
+            Add-FGTUserRADIUS -Name $pester_userradius -server $pester_userradiusserver1 -secret $pester_userradius_secret
+        }
+
+        It "Add User Local $pester_userlocal as RADIUS user" {
+            Add-FGTUserLocal -Name $pester_userlocal -radius_server $pester_userradius
+            $userlocal = Get-FGTUserLocal -name $pester_userlocal
+            $userlocal.name | Should -Be $pester_userlocal
+            $userlocal.type | Should -Be "radius"
```

Comment:
check also type and '.radius-server' for other test ? (and also for UserLocal ?)